### ISSUE TITLE
Unable to invoke 3-dots ellipsis sub-menu using keyboard

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#40239]
### Description

Unable to invoke 3-dots ellipsis sub-menu using keyboard

### Steps to reproduce

1. install `1.70.41`
2. launch Brave
3. restart Brave to pick up Griffin seeds for Playlist
4. load `https://www.youtube.com/watch?v=t7bQwwqW-Hc`
5. add to playlist
6. keep pressing `tab` to advance to it in `Playlist`
7. press `enter`/`return` whilst selected
8. while focused on the media in the sidebar, press `spacebar` or `return`/`enter`
9. try to do an action to invoke the `...` 3-dots ellipsis sub-menu

### Actual result

Unable to invoke the `...` sub-menu

<img width="381" alt="Screen Shot 2024-08-02 at 3 00 13 PM" src="https://github.com/user-attachments/assets/0c135f1a-a204-4d11-8b67-4ccadc04b453">

### Expected result

Should've been able to trigger and then navigate the `...` 3-dots sub-menu

<img width="393" alt="Screen Shot 2024-08-02 at 2 59 42 PM" src="https://github.com/user-attachments/assets/a10f2ea2-1fb3-49d1-9352-94e0b6e238bc">

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

```

Brave | 1.70.41 Chromium: 127.0.6533.88 (Official Build) nightly (x86_64)
-- | --
Revision | 9303bae9fe07bc2ec7c0cdef139e87304b2d6c96
OS | macOS Version 11.7.10 (Build 20G1427)
```

### Channel information

- [ ] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [X] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc @aguscruiz @rebron @sangwoo108 @brave/qa-team

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.71.96 Chromium: 129.0.6668.70 (Official Build) beta (64-bit)
-- | --
Revision | 6dc1837bd07e6dacd8b6b7b5d9f27e7bb09b5eb5
OS | Windows 11 Version 23H2 (Build 22631.4169)
```

1. install 1.71.96
2. launch Brave
3. enabled `#playlist` via `brave://flags`
4. load https://www.youtube.com/watch?v=DOKVREgWKbE
5. add to playlist
6. keep pressing tab to advance to it in Playlist
7. press enter/return whilst selected
8. while focused on the media in the sidebar, press spacebar or return/enter

Confirmed 3-dots ellipsis sub-menu was invoked when clicked keyboard return/enter

example | example 
----- | ---
![image](https://github.com/user-attachments/assets/660d4ba1-641d-4248-91d4-d79f038a53b3)|![image](https://github.com/user-attachments/assets/8838933f-7f66-4616-b6ec-a7ea2556157b)
